collect-adb.rst

```diff
@@ -0,0 +1,115 @@
+*****************************************
+Using Android Debug Bridge with Collect
+*****************************************
+
+This document focuses specifically on using **adb** as a command line tool to perform tasks related to Collect. 
+
+.. _what-is-adb:
+
+What is Android Debug Bridge (adb)?
+====================================
+
+Android Debug Bridge or `adb <https://developer.android.com/studio/command-line/adb.html>`_ is a command line tool which acts as a bridge between the Android device and the terminal. It can control device over USB from a computer, copy files back and forth, install and uninstall apps, run shell commands, and more. To install **adb**, please follow the instructions given `here <https://android.gadgethacks.com/how-to/android-basics-install-adb-fastboot-mac-linux-windows-0164225/>`_. For the developers and users of ODK Collect, the most common uses are:
+
+- include pushing blank forms to SD Card
+- pulling the form database, deleting forms
+- making the **.apk** file from the source code
```

Comment:
adb doesn't make the apk file, does it? It's used to load the apk file after make? (Right? I'm guessing, as I have not built Coolect from source)